### PR TITLE
As/dialog ui

### PR DESCRIPTION
**Pull request by Apsaliya**
Change images in both consent and rate us dialog ui and implement _later_ and _never_ options in rate us dialog.

### CODE REVIEW

File: app/src/main/res/layout/analytics_consent.xml [hunk 1]
```diff
@@ -36,10 +36,9 @@
     android:layout_marginEnd="@dimen/_12sdp"
     android:layout_marginStart="@dimen/_12sdp"
     android:layout_marginTop="@dimen/_12sdp"
-    android:checked="true"
+    android:checked="false"
```
Comment (Apsaliya) [inline app/src/main/res/layout/analytics_consent.xml]: GDPR!

File: app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt [hunk 2]
```diff
@@ -48,6 +54,24 @@ object RateUsHelper {
...
+      }
+
+      btNever.setOnClickListener {
+        onNoSelected(activity)
+        dialog.dismiss()
+      }
+
+      dialog.setOnKeyListener { _, keyCode, _ ->
+        if (keyCode == KEYCODE_BACK) {
+          onNoSelected(activity)
```
Comment (Hirak20) [inline app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt]: Should we consider back press as never? Maybelater would be appropriate in this case I think

File: rateus/src/main/java/com/example/rateus/RateusCore.kt [hunk 3]
```diff
@@ -65,10 +79,26 @@ object RateusCore {
     }
   }
 
-  fun onYesSelected(activity: Activity) {
+  private fun markInteractionDone(activity: Activity) {
```
Comment (Hirak20) [inline rateus/src/main/java/com/example/rateus/RateusCore.kt]: This method is same as onCancelSelected() ?

File: app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt [hunk 4]
```diff
@@ -30,12 +33,15 @@ object RateUsHelper {
       dialog.window.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
       val btClose = dialog.findViewById<ImageView>(R.id.ivCross)
       val btYes = dialog.findViewById<Button>(R.id.btnSave)
+      val btLater = dialog.findViewById<Button>(R.id.btnLater)
+      val btNever = dialog.findViewById<Button>(R.id.btnNever)
       btClose.setOnClickListener {
         onCancelSelected(activity)
         dialog.dismiss()
```
Comment (Hirak20) [inline app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt]: Same as backpress. I think we can keep interaction as false when back pressed or close is clicked

File: app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt [hunk 5]
```diff
@@ -30,12 +33,15 @@ object RateUsHelper {
       dialog.window.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
       val btClose = dialog.findViewById<ImageView>(R.id.ivCross)
       val btYes = dialog.findViewById<Button>(R.id.btnSave)
```
Comment (Hirak20) [inline app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt]: Can use views directly from Kotlin Android Extensions as we have contentview. No need for findviewbyid

**Comment by Hirak20** [comment]
Consent dialog will have to be non cancelable on back press as well. Need to add cancelable(false)

**Pull request merged**